esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -312,20 +315,13 @@ DATABASE_NAME = '$DBNAME'
 DATABASE_HOST = 'localhost'
 DATABASE_PORT = '5432'
 
+VARNISH_HOST = 'localhost'
+VARNISH_PORT = '80'
+
 from database_settings import *
 
 MIDDLEWARE_LOCAL = []
-
-# E-mails for contact form
-email_choices = (
-    ('general', 'General Inquiries'),
-    ('web',     'Web Site Problems'),
-    )
-# Corresponding email addresses                                                                                                                                 
-email_addresses = {
-    'general': '$GROUPEMAIL',
-    'web':     '$SITENAME-websupport@learningu.org',
-    }
+SECRET_KEY = '`${CURDIR}/random_password.sh`'
```

Comment:
Is there a reason we're moving towards having the default configs inline rather than in a separate file?  Now might be a nice time to move away from that, which I think we've discussed doing.